apps/docs/content/components/OrderedList.mdx

```diff
@@ -0,0 +1,44 @@
+---
+title: List
+status: Experimental
+figma: 'https://www.figma.com/file/BJ95AjraesmRCWsKA013GS/Primer-Brand?node-id=354%3A9513'
```

Comment:
This goes to typography page in the Figma library 🤔

Reply:
In next push